checkout-ui-custom/src/partials/Deliver/DeliveryOptions.js

```diff
@@ -1,14 +1,14 @@
 import Radio from './Elements/Radio';
 
-const DeliveryOptions = () => `
+const DeliveryOptions = ({ hasFurnOnly }) => `
   <label class="bash--delivery-option-display" >
   ${Radio({
     name: 'delivery-option',
     options: [{ checked: true, value: true }],
   })}
    
    <div id="bash--delivery-option-text" class="bash--delivery-option-text">
-      <span class="normal-delivery">Deliver within 3 - 5 days</span>
+      <span class="normal-delivery">Deliver within ${hasFurnOnly ? '5 - 10' : '3 - 10'} working days</span>
```

Comment:
We still need a non-furniture scenario here - `3 - 5 working days` for non-furniture.

Reply:
updated this